File: Data Structures & Algorithms/single-number/submission-2.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    singleNumber(nums) {
        for (let i = 0; i < nums.length; i++) {
            let flag = true;
            for (let j = 0; j < nums.length; j++) {
                if (i !== j && nums[j] === nums[i]) {
                    flag = false;
                }
            }
            if (flag) return nums[i];
        }
    }
}

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-2.py
class Solution:
    def productExceptSelf(self, nums: List[int]) -> List[int]:
        n = len(nums)
        ans = [0]*n
        for i in range(n):
            final = 1
            for j in range(n):
                if i != j:
                    final *= nums[j]
            ans[i] = final
        return ans